go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/fullstorydev/emulators
+
+go 1.13
+
+require (
```

Comment:
added a module on the top project, if not we would need to install the following modules in both emulators, I think this way its simplier

Reply:
I don't think this is a great idea, we don't want people to erroneously `go get github.com/fullstorydev/emulators`.

Reply:
@jhump how do you do this in other projects where you potentially need some build-time tools to do things like run static checks, but you don't want those go deps to appear in your project's go.mod because the static check tools are completely unrelated to your project?

Reply:
> you don't want those go deps to appear in your project's go.mod

Does not compute. That's the way you pin the versions for them. The `go.mod` pins _all_ dependencies, not just compile-time dependencies of the code.

In [other repos](https://github.com/fullstorydev/grpcurl/blob/master/go.mod#L7), they are simply in the `go.mod` file, but annotated (by `go get -d ...` when the tool versions were first fetched/defined) as `// indirect`.

What I _want_ to do in all of the repos is to add a `tools.go` file. This is [a pattern](https://marcofranssen.nl/manage-go-tools-via-go-modules#tldr) where you have a `tools.go` file that has build constraints so it is never actually built/compiled. Its presence is primarily to tell the Go tool chain that you depend on these tools; in other words, it prevents `go mod tidy` from deleting `go.mod` records for these tools.

Reply:
I like the idea, but I'd want that in like a tools sub folder